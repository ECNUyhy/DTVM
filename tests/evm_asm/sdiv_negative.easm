// SDIV(-5, 2) = -2（负/正 → 负，向零截断）
PUSH1 0x02                              // B = 2
PUSH32 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffb  // A=-5
SDIV
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
