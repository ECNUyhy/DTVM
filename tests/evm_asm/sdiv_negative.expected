status: success
error_code: 0
stack: []
memory: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE'
storage: {}
transient_storage: {}
return: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE'
events: []